java/ql/test/library-tests/frameworks/android/flow-steps/test.ql

```diff
@@ -0,0 +1,53 @@
+import java
+import semmle.code.java.dataflow.DataFlow
+import semmle.code.java.dataflow.ExternalFlow
+import semmle.code.java.dataflow.TaintTracking
+import TestUtilities.InlineExpectationsTest
```

Comment:
The boilerplate here has been abstracted into `java/ql/test/TestUtilities/InlineFlowTest.qll` so just import that.